httpbin/handlers_test.go

```diff
@@ -33,21 +33,28 @@ import (
 const (
 	maxBodySize int64         = 1024
 	maxDuration time.Duration = 1 * time.Second
+	testPrefix                = "/a-prefix"
 )
 
-// "Global" test app, server, & client to be reused across test cases.
-// Initialized in TestMain.
-var (
+type environment struct {
 	app    *HTTPBin
```

Comment:
Seems like we don't actually need this `app` field

Reply:
You are right. I removed it. Was originally needed when there was no test server.